sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1048,6 +1048,16 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val DECIMAL_OPERATIONS_ALLOW_TRUNCAT =
+    buildConf("spark.sql.decimalOperations.allowTruncat")
```

Comment:
`allowTruncat` -> `allowTruncate`?

Reply:
thanks. This is what @cloud-fan suggested. Actually we are not truncating, but rounding. Personally, I'd prefer `allowPrecisionLoss`, but I am fine with any value. WDYT @cloud-fan @gatorsmile?

Reply:
Sorry that was my typo... `allowPrecisionLoss` SGTM